Resources/Prototypes/Floof/Entities/Clothing/Shoes/Combatmags.yml

```
@@ -0,0 +1,14 @@
+- type: entity
+  parent: ClothingShoesBootsMagAdv
+  id: ClothingShoesBootsMagCom
+  name: Combat magboots
+  description: Made by SESWC. These State-of-the-art magnetic boots that do not slow down their wearer with built in armor for extra protection
```

```suggestion
  description: Made by SESWC. These State-of-the-art magnetic boots with built in armor for extra protection
```
Edit description as needed.